pip/req/req_install.py

```diff
@@ -312,7 +312,13 @@ def build_location(self, build_dir):
             # package is not available yet so we create a temp directory
             # Once run_egg_info will have run, we'll be able
             # to fix it via _correct_build_location
-            self._temp_build_dir = tempfile.mkdtemp('-build', 'pip-')
+            #
+            # NOTE: Some systems have /tmp as a symlink which confuses
```

Comment:
Since the above comment is already a note, it makes little sense to demarkate this paragraph as one.